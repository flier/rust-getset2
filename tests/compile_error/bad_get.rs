use getset2::Getter;

#[derive(Getter)]
#[get(foobar)]
struct Foobar {}

fn main() {}
