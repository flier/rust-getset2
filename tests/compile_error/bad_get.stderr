error: invalid #[get(..)] attribute, cannot find parameter `foobar` in this scope
 --> tests/compile_error/bad_get.rs:4:1
  |
4 | #[get(foobar)]
  | ^
